### PR TITLE
reorganize manifests

### PR DESCRIPTION
**Pull request by kukushking**
## Describe your changes
1. Main manifest getting XXL. Reorganize manifests per usecase:
- mlops-sagemaker
- mwaa-ml-training
- mlflow-experiments-tracking

2. Add an uber-manifest that deploys all modules.
3. Move event bus for consistency group/module.

## Checklist before requesting a review

- [x] I updated `CHANGELOG.MD` with a description of my changes
- [x] If the change was to a module, I ran the code validation script (`scripts/validate.sh`)
- [x] If the change was to a module, I have added thorough tests
- [x] If the change was to a module, I have added/updated the module's README.md
- [x] If a module was added, I added a reference to the module to the repository's README.md
- [x] I verified that my code deploys successfully using `seedfarmer apply`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: AIOpsModuleTestsInfrastruct-doMLXEYsnmxr
* Commit ID: 108add5dfea9113810a6609bfcc73048cc2c3c67
* Result: SUCCEEDED
* [Build Logs](https://ythfngxmh6.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=13739ce0-0628-413e-84cd-cd6b421d864b%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: AIOpsModuleTestsInfrastruct-doMLXEYsnmxr
* Commit ID: 16c6cb15969e65b729102247b8c967abf881c5b9
* Result: SUCCEEDED
* [Build Logs](https://ythfngxmh6.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=02f1ed49-1f50-40e0-b171-ef9c738a496c%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: AIOpsModuleTestsInfrastruct-doMLXEYsnmxr
* Commit ID: 3d7da89ce084c56d7bb7e15e3542d96fa26f77f7
* Result: SUCCEEDED
* [Build Logs](https://ythfngxmh6.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=ce2d716f-fd6f-413a-a52c-c104118744b4%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: AIOpsModuleTestsInfrastruct-doMLXEYsnmxr
* Commit ID: 667bb578d70b9e6bbc05872bd8aca9d11e4c1500
* Result: SUCCEEDED
* [Build Logs](https://ythfngxmh6.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=f213faab-5992-4a0b-8aa8-55266a113879%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: AIOpsModuleTestsInfrastruct-doMLXEYsnmxr
* Commit ID: 90e988338a4c976185a00230b29fca4158d35b27
* Result: SUCCEEDED
* [Build Logs](https://ythfngxmh6.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=77b2a0d0-193a-490d-8231-0c49881e5d6c%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: manifests/uber-deployment.yaml [hunk 1]
```diff
@@ -29,5 +29,5 @@ targetAccountMappings:
         envVariable: PRIMARY_ACCOUNT
     default: true
     regionMappings:
-      - region: us-east-1
-        default: true
\ No newline at end of file
+      - region: us-east-2
```
[the remark this answers is not in the document]
Comment (kukushking) [reply manifests/uber-deployment.yaml]: Nope, thanks for noticing

**Review by LeonLuttenberger — APPROVED** [review]

**Pull request merged**